tests/specs/functional/lemmas-spec.k

```diff
@@ -13,6 +13,13 @@ module VERIFICATION
  // --------------------------------------
     rule runLemma( T ) => doneLemma( T )
 
+ // Shorthand for previous notations so that more complex
+ // claims do not have to be rewritten
+
+    syntax Bytes ::= Bytes "[" Int ".." Int "]" [function, total]
+ // -------------------------------------------------------------
+    rule B:Bytes [ S:Int .. W:Int ] => #range(B, S, W)
+
 endmodule
```

Comment:
Lookin ghrough, I only see a handful of new lemmas using the shorthand, can we just simplify to the `#range(...)` syntax those lemmas? No need to give future-selves the option, which will lead to more confusion for future new people.